### PR TITLE
chore(redoc): update javascript dependencies (and fix script to update them)

### LINKED ISSUE #4354: Strange rendering bug with ReDoc and iri-references
After browsing several API documentations, all based on API Platform and using ReDoc UI, I've found some strange rendering bug:

**Description**  
image

All links in `hydra:view` context are generated with `../словник` value.

This does not occur using SwaggerUI and I did not find any issue mentionning this on [ReDoc issues](https://github.com/Redocly/redoc/issues).

**How to reproduce**  

A simple Google search on `api platform "/словник"`

- https://fashion.plus.demo.sylius.com/api/v2?ui=re_doc#operation/postCredentialsItem
- https://demo.api-platform.com/docs?ui=re_doc#tag/Book
- https://api.mail.tm/docs?ui=re_doc#tag/Domain
- https://api.cwa.rocks/docs?ui=re_doc

image

### LINKED ISSUE #4394: Redoc use Ukranian language
**API Platform version(s) affected**: 2.6

**Description**  
https://demo.api-platform.com/docs?ui=re_doc#tag/Book
=> 
```
"hydra:view": {
"@id": "../словник",
"@type": "string",
"hydra:first": "../словник",
"hydra:last": "../словник",
"hydra:previous": "../словник",
"hydra:next": "../словник"
},
```

It would be better to use plain English by default

### PR DESCRIPTION
**Pull request by lyrixx**
fixes #4394
fixes #4354 
And it also fixe other bugs

### CODE REVIEW

**Comment by dunglas** [comment]
Can you run `update-js.sh` instead to update the `yarn.lock` file please? It's used by the vulnerability scanner of GitHub.

**Comment by lyrixx** [comment]
@dunglas ✅

File: update-js.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ rm -f yarn.lock
 yarn add @fontsource/open-sans swagger-ui-dist
 yarn add --production es6-promise fetch react react-dom graphiql graphql-playground-react@1.7.26 redoc
 
-dest=src/Bridge/Symfony/Bundle/Resources/public/fonts/open-sans/
```
Comment (dunglas) [inline update-js.sh:7]: This is probably not wanted. cc @soyuka

Comment (soyuka) [reply update-js.sh:7]: np for now I'll move this later

**Comment by soyuka** [comment]
the commit should be something like: `chore(redoc): update javascript dependencies`

File: update-js.sh (hunk 1 above)
Comment (lyrixx) [reply update-js.sh:7]: It's required, since everything has been moved

**Comment by lyrixx** [comment]
I updated the commit and PR message

**Comment by soyuka** [comment]
thanks @lyrixx

**Pull request merged**